test/syscalls/linux/futex.cc

```diff
@@ -694,7 +694,7 @@ TEST_P(PrivateAndSharedFutexTest, PIWaiters) {
 
   // Wait until the thread blocks on the futex, setting the waiters bit.
   auto start = absl::Now();
-  while (a.load() != (FUTEX_WAITERS | gettid())) {
+  while (a.load() != int(FUTEX_WAITERS | gettid())) {
```

Comment:
According to the linter, this casting is considered deprecated. We are expected to use static_cast<int>(...).